lib/src/command_runner.dart

```diff
@@ -5,6 +5,7 @@ import 'package:mason/mason.dart' hide packageVersion;
 import 'package:pub_updater/pub_updater.dart';
 import 'package:usage/usage_io.dart';
 import 'package:very_good_cli/src/commands/commands.dart';
+import 'package:very_good_cli/src/commands/create/create_legacy.dart';
```

Comment:
do we need this additional import or can it be added to the commands.dart barrel?